vault/identity_store_oidc_provider.go

```diff
@@ -356,6 +369,14 @@ func oidcProviderPaths(i *IdentityStore) []*framework.Path {
 		},
 		{
 			Pattern: "oidc/provider/?$",
+			Fields: map[string]*framework.FieldSchema{
+				"allowed_client_id": {
+					Type: framework.TypeString,
```

Comment:
Not sure if this is a valid use case (and don't think it's part of the requirements, so feel free to ignore), but would someone ever want to provide a more than one `client_id` to filter?

Reply:
Good question -- just checked with UI/UX team on this. They don't have a need for providing more than one `client_id` to filter on. Going to resolve this one.